docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -0,0 +1,164 @@
+
+.. include:: /getting-started/github-apps/note--vocabulary.rst
+
+Install the Dockstore GitHub App
+--------------------------------
+.. include:: /getting-started/github-apps/snippet--installation.rst
+
+Publishing Your Notebook
+------------------------
+In this release, you are only able to publish your notebook using the ``publish`` key in the .dockstore.yml file, and your notebook must be published in order for you to view it on Dockstore.
+
+Publishing will create a public page that is very similar to the ones we have for tools and workflows.
+
+.. figure:: /assets/images/docs/notebooks-public-page.png
+
+Now other users can view and :doc:`star <../end-user-topics/starring>` your notebook. You will also have the option to add published notebooks to a :doc:`collection <../advanced-topics/organizations-and-collections>`.
+
+Saving Changes to Your Notebook on Google Colab to GitHub
+---------------------------------------------------------
+
+At this point, your Google Colab and GitHub integration should be set up, you have installed the Dockstore GitHub App to your GitHub repository containing your notebook, and the published notebook appears on Dockstore. 
+
+You may wish to continue developing your notebook on Google Colab by modifying and re-running it. In order for the changes to your notebook to appear on Dockstore, you must click on File > Save a copy in GitHub to save your notebook to your GitHub repository.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github.png
+    :width: 35 %
+
+Configure where you want to save the notebook to on GitHub then click OK. Google collab will push a commit to your repository with the notebook.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github-dialog.png
+    :width: 60 %
+
+Dockstore will detect the new commit and automatically sync your notebook from GitHub to Dockstore.
+
+Launch With Google Colab
+------------------------
+
+When viewing a public notebook on Dockstore, you can launch the notebook into Google Colab by clicking on the Google Colab link on the Launch With panel located on the right side of the page. This will open the notebook in Google Colab.
+
+.. figure:: /assets/images/docs/launch-with-google-colab.png
+
+Using the Notebooks Feature Flag
+--------------------------------
+
+There is a ``notebooks`` feature flag that can be used to explore Dockstore Notebooks feature that are in development.
+
+.. warning:: The features hidden behind this flag may not be fully functional, but using the flag offers you a preview of the features to come.
+
+To use the ``notebooks`` feature flag, append ``notebooks`` to the Dockstore URL as a query parameter. You only need to do this once, unless you refresh/close your browser.
+
+For example, if you're on the ``https://dockstore.org`` page, append ``notebooks`` such that it looks like this: ``https://dockstore.org?notebooks``.
```

Comment:
Can make the examples here links, i.e. ``https://dockstore.org?notebooks`` should be an active link like [https://dockstore.org?notebooks](https://dockstore.org?notebooks)